src/cargo/util/workspace.rs

```diff
@@ -22,7 +22,8 @@ fn get_available_targets<'a>(
         .map(Target::name)
         .collect();
 
-    targets.sort();
+    // sorting primitive type str
```

Comment:
GIven that the list here is probably always quite small is using a faster sort to lose readability really worth it?

Reply:
I don't think readability is worse but I reverted the change. Will push once nr1 is resolved